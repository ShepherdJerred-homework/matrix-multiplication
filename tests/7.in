3 -4 4 8 4 7 -3 3 6 -2 5 -3 8 -7 6 -9 -5 -5 3 2 5 8 6 -7 5 -10 7 7 -4 -2 -5 4 -3 4 -3 -9 -1 9 3 4 -1 -9 5 -2 -4 -2 -10 -1 1 -1 9 -1 2 9 0 -5 -1 10 -4 5 9 8 -3 8 10 -5 -4 1 2 -6 2 7 0 1 7 1 8 -3 -9 -1 -9 -3 2 7 9 -4 -8 5 9 7 4 -10 -9 -9 -3 5 -7 8 1 5 1 -3 9 5 6 9 6 -9 -4 0 -2 9 5 1 4 2 -8 -4 -3 7 -6 -8 -5 8 -5 4 1 -8 -5 6 -9 2 2 6 10 9 -8 -8 9 -4 -6 -3 6 -9 2 -2 -6 -10 -1 -6 9 -4 -9 5 7 2 -6 10 6 10 -7 5 -7 8 -9 1 10 10 -1 -6 8 5 -2 6 0 3 -3 10 -2 -2 -5 -2 2 7 -5 -9 -6 -7 3 -8 5 7 -2  8 1 7 