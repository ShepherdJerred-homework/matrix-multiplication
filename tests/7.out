28
18
16
93
-113
-24
77
76
-24
-69
57
-40
103
18
-34
-74
-6
25
29
36
117
47
-32
23
22
-2
-38
86
-5
107
-152
-68
100
68
109
11
61
69
-20
-59
-13
-29
-27
-56
121
-9
-59
53
-92
49
-6
16
156
-100
62
83
-5
-10
91
-53
35
-91
-109
33
